modules/ROOT/pages/enabling-sso-for-your-portal.adoc

```diff
@@ -116,11 +117,42 @@ Before enabling SSO, ensure you have the following permissions and context:
 * Experience with identity management and setting up identity providers for SSO.
 * Review Access Management's xref:access-management::external-identity.adoc[Identity Provider] documentation to understand identity management for Anypoint Platform.
 
+* Be aware of what API Experience Hub expects for claims or attributes when creating a new application for the portal in the identity provider.
++
+Configuring the user claims in the identity provider ensures that the user information is mapped correctly to Salesforce and Anypoint users.  
++
+The AEHPortalRegistrationHandler requires specific user information from the identity provider to create or update a user. The following tables list the required information and which claims the information is extracted from. If there is more than one claim for the same value, the registration handler looks for the first expected claim in the order it appears in the following tables:
```

Comment:
```suggestion
The AEHPortalRegistrationHandler requires specific user information from the identity provider to create or update a user. The following tables list the required information and expected claims. If more than one claim exists for the same value, the registration handler uses the first expected claim in the order it appears in the following tables:
```

Reply:
I think it would be helpful to explain what you mean by "expected claim" (the thing the handler extracts user information from?) earlier in this description. Then, you can just call it expected claim.

Reply:
Should that be "following table" (or does this sentence also apply to the second table)? I ask because the second table doesn't mention claims.

Reply:
That is a good idea to explain what is meant by expected claims. I will add that.